applications/nrf_desktop/configuration/common/led_state.h

```diff
@@ -4,45 +4,66 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
+/** @file
+ * @brief Header file with LED states.
+ */
+
 #ifndef _LED_STATE_H_
 #define _LED_STATE_H_
 
-/**
- * @brief LED State
- * @defgroup led_state LED State
- * @{
- */
-
 #include <zephyr/types.h>
 #include <caf/events/led_event.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+/**
+ * @brief LED State
+ * @defgroup nrf_desktop_led_state LED State
+ * @{
+ */
+
+/** @brief System states represented by LED. */
 enum led_system_state {
+	/** Device is idle. */
 	LED_SYSTEM_STATE_IDLE,
+	/** Device is charging. */
 	LED_SYSTEM_STATE_CHARGING,
+	/** Device in error state. */
 	LED_SYSTEM_STATE_ERROR,
 
+	/** Number of system states. */
 	LED_SYSTEM_STATE_COUNT
 };
 
+/** @brief Peer states represented by LED. */
 enum led_peer_state {
+	/** Peer is disconnected. */
 	LED_PEER_STATE_DISCONNECTED,
+	/** Peer is connected. */
 	LED_PEER_STATE_CONNECTED,
+	/** Device searches for peer. */
 	LED_PEER_STATE_PEER_SEARCH,
+	/** Device waits for user to confirm peer switch. */
 	LED_PEER_STATE_CONFIRM_SELECT,
+	/** Device waits for user to confirm bond erase. */
 	LED_PEER_STATE_CONFIRM_ERASE,
+	/** Device searches for new peer before bond erase. */
 	LED_PEER_STATE_ERASE_ADV,
 
+	/** Number of peer states. */
 	LED_PEER_STATE_COUNT
 };
 
-enum led_id {
+/** @brief LED identification. */
+enum led_id_nrf_desktop {
```

Comment:
@divipillai , I added postfix to the name. It's not pretty but should do the trick.